README.md

```diff
@@ -121,7 +118,7 @@ const result = await codeClient.extendBundle({
   },
   removedFiles: [],
 });
-const { bundleId, missingFiles, uploadURL } = result;
```

Comment:
why are we removing the `uploadURL` attribute from docs?

Reply:
it was deprecated and not used by anyone for a very long time. So it's just a clean-up